pkg/logs/pipeline/provider.go

```diff
@@ -54,14 +55,16 @@ func (pp *PipelineProvider) Start(cm *sender.ConnectionManager, auditorChan chan
 	}
 }
 
-func (pp *PipelineProvider) MockPipelineChans() {
+// MockPipelineChans initializes pipelinesChans for testing purpose
+// TODO: move this somewhere else
```

Comment:
where would you move this? I'm not sure how to do that without exposing private attributes

Reply:
I guess we could create another file with `_test` suffix

Reply:
Let's do it then!